pkg/test/framework/components/istio/flags.go

```diff
@@ -41,4 +41,8 @@ func init() {
 		e.g. components.cni.enabled=true,components.cni.namespace=kube-system`)
 	flag.BoolVar(&settingsFromCommandline.EnableCNI, "istio.test.istio.enableCNI", settingsFromCommandline.EnableCNI,
 		"Deploy Istio with CNI enabled.")
+	flag.StringVar(&settingsFromCommandline.IngressGatewayServiceName, "istio.test.kube.ingressgatewayServiceName",
```

Comment:
Because the original kebab-case is `istio-ingressgateway`, I kept "ingressgateway" as one word for the flag, but I realize I didn't do it for the struct field....not sure if the codebase has strong opinions. I see both used

Reply:
Ah, I see what you're saying. Fixed!